From FEDORA_PATCHES Mon Sep 17 00:00:00 2001
From: Keith Seitz <keiths@redhat.com>
Date: Tue, 28 Jul 2020 09:49:56 -0400
Subject: gdb-rhbz1842691-corefile-mem-access-15of15.patch

;; Add period to help text for maint print core-file-backed-mappings
;; Kevin Buettner, RH BZ 1842961

   Author: Kevin Buettner <kevinb@redhat.com>
   Date:   Thu Jul 23 13:26:44 2020 -0700

    Fix BZ 26294 - Add period to help text for maint print core-file-backed-mappings

    gdb/ChangeLog:

    	PR corefiles/26294
    	* corelow.c (_initialize_corelow): Add period to help text
    	for "maintenance print core-file-backed-mappings".

diff --git a/gdb/corelow.c b/gdb/corelow.c
--- a/gdb/corelow.c
+++ b/gdb/corelow.c
@@ -1388,6 +1388,6 @@ Show whether CORE-FILE loads the build-id associated files automatically."),
 			   &setlist, &showlist);
   add_cmd ("core-file-backed-mappings", class_maintenance,
            maintenance_print_core_file_backed_mappings,
-	   _("Print core file's file-backed mappings"),
+	   _("Print core file's file-backed mappings."),
 	   &maintenanceprintlist);
 }
